content-services/latest/tutorial/sso/saml.md

```diff
@@ -0,0 +1,351 @@
+| oauth2.redirectUriLogout       | APP_CONFIG_OAUTH2_REDIRECT_LOGOUT            | The URL to redirect to after a user successfully signs out                                                                                                                                                                                                   |
+
+> **Note:** If `implicitFlow` is set to `false` the grant type `password` will be used instead.
+
+The following is an example `app.config.json` file excerpt. By default this file is located in the `/src` directory.
+
+```json
+"authType": "OAUTH",
+"oauth2": {
+        "host": "https://keycloak.example.com/auth/realms/alfresco",
+        "clientId": "alfresco",        
+        "scope": "openid",
+        "implicitFlow": true,
+        "silentLogin": true,
+        "redirectSilentIframeUri": "https://adw.example.com/workspace/assets/silent-refresh.html",
+        "redirectUri": "/workspace/",
+        "redirectUriLogout": "/workspace/logout"
+        }
+```
+
+## Step 8: Configure Alfresco Share properties
+
+The properties listed that need to be set for Alfresco Share are only those that are required for setting up SSO.
+
+Use the following configuration parameters either in the `share-config.properties` file, using the share config map in Kubernetes, or as environment variables in a Docker Compose file:
+
+|Property|Description|
+|--------|-----------|
+| aims.enabled | Enables or disables Keycloak, for example `true`. |
+| aims.realm | The name of the realm, for example `alfresco`. |
+| aims.resource | The Client ID of the application, for example `share`. |
+| aims.authServerUrl | The base URL of the Keycloak, for example `https://keycloak.example.com` |
+| aims.publicClient | If set to `true`, the adapter will not send credentials for the client to Keycloak. |
+
+## Step 9: (Optional) Configure Alfresco Sync Service
+
+If Alfresco Sync Service is used and a client has been created for it in [step 1](#step-1-configure-a-realm-and-clients) then the following properties need to be set in the `sync/service-sync/config.yml`:
+
+| Property | Description |
+| -------- | ----------- |
+| keycloak.auth-server-url |The base URL of the Keycloak, for example `https://keycloak.example.com/auth`|
+| keycloak.realm |The realm name configured in the Keycloak for the Alfresco application, for example `alfresco`|
+| keycloak.resource |The **Client ID** set up in the Desktop Sync for Alfresco Content Services. The client needs to exist underneath the realm set for `keycloak.realm`, for example `desktop-sync`|
+| keycloak.public-client |The adapter will not send credentials for the client to the Keycloak if this is set to true, for example `true`|
+| keycloak.credentials.secret |The secret key for this client if the access type is not set to public.|
```

Comment:
```suggestion
| identity-service.auth-server-url |Keycloak's base URL, for example `https://keycloak.example.com/auth`|
| identity-service.realm |The realm name configured in Keycloak for the Alfresco application, for example `alfresco`|
| identity-service.resource |The **Client ID** set up in the Desktop Sync for Alfresco Content Services. The client needs to exist underneath the realm set for `identity-service.realm`, for example `desktop-sync`|
| identity-service.public-client |The adapter will not send credentials for the client to Keycloak if this is set to true, for example `true`|
| identity-service.credentials.secret |The secret key for this client if the access type is not set to public.|
```

Again, these property keys can't be updated to something that doesn't reflect the current status of the software. They have to be reverted to their original variant.